Actions failed

```
     Error: Spawning of process failed. (Error was: Inappropriate ioctl for device
        ... Additional info: "Could not find command: \'/home/runner/.choosenim/toolchains/nim-1.4.8/bin/nim\'. OS error: Permission denied")
      Info: If unexpected, please report this error to https://github.com/dom96/choosenim
Error: Process completed with exit code 1.
```

https://github.com/jiro4989/setup-nim-action/runs/3786283506?check_suite_focus=true

--- a/lib/installer.js
+++ b/lib/installer.js
@@ -48,13 +48,13 @@ function setNimbleBinPath() {
 }
 function installNim(version, noColor, yes) {
     return __awaiter(this, void 0, void 0, function* () {
-        const body = yield request({
+        let body = yield request({
             url: 'https://nim-lang.org/choosenim/init.sh',
             method: 'GET',
         });
+        body = body.replace(/\$stable_version/g, '0.7.4'); // #62
         fs.writeFileSync('init.sh', body);
         process.env.CHOOSENIM_NO_ANALYTICS = '1';
-        process.env.CHOOSENIM_CHOOSE_VERSION = '0.7.4'; // #62
         // #38
         if (util.isGlobPatchVersion(version) || util.isGlobMinorVersion(version)) {
             core.info(`Fetch a latest versions with ${version}`);

--- a/lib/installer.js
+++ b/lib/installer.js
@@ -54,7 +54,7 @@ function installNim(version, noColor, yes) {
         });
         fs.writeFileSync('init.sh', body);
         process.env.CHOOSENIM_NO_ANALYTICS = '1';
-        process.env.CHOOSE_VERSION = 'v0.7.4'; // #62
+        process.env.CHOOSENIM_CHOOSE_VERSION = 'v0.7.4'; // #62
         // #38
         if (util.isGlobPatchVersion(version) || util.isGlobMinorVersion(version)) {
             core.info(`Fetch a latest versions with ${version}`);

--- a/lib/installer.js
+++ b/lib/installer.js
@@ -54,7 +54,7 @@ function installNim(version, noColor, yes) {
         });
         fs.writeFileSync('init.sh', body);
         process.env.CHOOSENIM_NO_ANALYTICS = '1';
-        process.env.CHOOSENIM_CHOOSE_VERSION = 'v0.7.4'; // #62
+        process.env.CHOOSENIM_CHOOSE_VERSION = '0.7.4'; // #62
         // #38
         if (util.isGlobPatchVersion(version) || util.isGlobMinorVersion(version)) {
             core.info(`Fetch a latest versions with ${version}`);

--- a/lib/installer.js
+++ b/lib/installer.js
@@ -54,6 +54,7 @@ function installNim(version, noColor, yes) {
         });
         fs.writeFileSync('init.sh', body);
         process.env.CHOOSENIM_NO_ANALYTICS = '1';
+        process.env.CHOOSE_VERSION = 'v0.7.4'; // #62
         // #38
         if (util.isGlobPatchVersion(version) || util.isGlobMinorVersion(version)) {
             core.info(`Fetch a latest versions with ${version}`);

--- a/src/installer.ts
+++ b/src/installer.ts
@@ -22,7 +22,7 @@ async function installNim(version: string, noColor: boolean, yes: boolean) {
   })
   fs.writeFileSync('init.sh', body)
   process.env.CHOOSENIM_NO_ANALYTICS = '1'
-  process.env.CHOOSE_VERSION = 'v0.7.4' // #62
+  process.env.CHOOSENIM_CHOOSE_VERSION = 'v0.7.4' // #62
 
   // #38
   if (util.isGlobPatchVersion(version) || util.isGlobMinorVersion(version)) {

--- a/src/installer.ts
+++ b/src/installer.ts
@@ -22,7 +22,7 @@ async function installNim(version: string, noColor: boolean, yes: boolean) {
   })
   fs.writeFileSync('init.sh', body)
   process.env.CHOOSENIM_NO_ANALYTICS = '1'
-  process.env.CHOOSENIM_CHOOSE_VERSION = 'v0.7.4' // #62
+  process.env.CHOOSENIM_CHOOSE_VERSION = '0.7.4' // #62
 
   // #38
   if (util.isGlobPatchVersion(version) || util.isGlobMinorVersion(version)) {

--- a/src/installer.ts
+++ b/src/installer.ts
@@ -20,7 +20,7 @@ async function installNim(version: string, noColor: boolean, yes: boolean) {
     url: 'https://nim-lang.org/choosenim/init.sh',
     method: 'GET',
   })
-  body = body.replace(/\$stable_version/, '0.7.4') // #62
+  body = body.replace(/\$stable_version/g, '0.7.4') // #62
   fs.writeFileSync('init.sh', body)
   process.env.CHOOSENIM_NO_ANALYTICS = '1'
 

--- a/src/installer.ts
+++ b/src/installer.ts
@@ -16,13 +16,13 @@ function setNimbleBinPath() {
 }
 
 async function installNim(version: string, noColor: boolean, yes: boolean) {
-  const body = await request({
+  let body = await request({
     url: 'https://nim-lang.org/choosenim/init.sh',
     method: 'GET',
   })
+  body = body.replace(/\$stable_version/, '0.7.4') // #62
   fs.writeFileSync('init.sh', body)
   process.env.CHOOSENIM_NO_ANALYTICS = '1'
-  process.env.CHOOSENIM_CHOOSE_VERSION = '0.7.4' // #62
 
   // #38
   if (util.isGlobPatchVersion(version) || util.isGlobMinorVersion(version)) {

--- a/src/installer.ts
+++ b/src/installer.ts
@@ -22,6 +22,7 @@ async function installNim(version: string, noColor: boolean, yes: boolean) {
   })
   fs.writeFileSync('init.sh', body)
   process.env.CHOOSENIM_NO_ANALYTICS = '1'
+  process.env.CHOOSE_VERSION = 'v0.7.4' // #62
 
   // #38
   if (util.isGlobPatchVersion(version) || util.isGlobMinorVersion(version)) {

